Collaboration.md

```diff
@@ -14,7 +14,7 @@ Ways the project can choose to interact with the community:
 
 1. Review the [guide](https://bitcoin.design/guide/) to find a direction or solution
 1. Engage in Discord conversations to share ideas and get feedback
-1. Participate in a design review session
+1. Participate in a [design review session](Design-Reviews.md)
 
```

Comment:
1. Participate in a UX research review session

Reply:
Do we have something more official on UX review sessions? Can we also accommodate those in the design reviews to avoid having similar types of sessions?